chess/engine.py

```diff
@@ -281,7 +281,7 @@ def is_managed(self) -> bool:
 
 
 class Limit:
-    """Search termination condition."""
+    """Searches termination condition."""
```

Comment:
Looks good to me except this one. `Limit` does not search anything, it *is* the search termination codition.